tax_determination.rb

```diff
@@ -0,0 +1,17 @@
+module Tax_Determination
+  PRODUCT = %w(pills book books chocolate chocolates)
+  def isExempt(arr)
```

Comment:
Methods that return boolean values should end with `?`. And use `snake_case` for method and variable names. Use meaning full parameter names.
```suggestion
  def is_exempt?(products)
```

Reply:
I think this method should handle one product at a time since you are calling this method for each product.